gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -347,20 +347,39 @@
   public static final HadoopConfigurationProperty<Boolean> GCS_GRPC_ENABLE =
       new HadoopConfigurationProperty<>("fs.gs.grpc.enable", false);
 
-  /** Configuration key for enabling checksum validation for the gRPC API. */
+  /**
```

Comment:
Please re-format w/ https://github.com/google/google-java-format

Reply:
Code was formatted with `google-java-format` intellij plugin. 

The style guide indicates that the preferred style is multi line block [comments](https://google.github.io/styleguide/javaguide.html#s4.8.6-comments) and in case of [javadoc](https://google.github.io/styleguide/javaguide.html#s7.1.1-javadoc-multi-line), both are acceptable.

Reverting manually, we can verify the preferred format and re-format the files separately